src/en/developer-actions.md

```diff
@@ -0,0 +1,390 @@
+Title: implementing actions in juju charms
+
```

Comment:
I (wrongly) assumed your scan of 'awesomeness' included correcting any legacy errors...

BTW, this happens to me all the time - I correct some error in a page of docs and then @petermatulis won't let me merge it until I fix all the other errors in the doc